css/style.css

```diff
@@ -1,5 +1,5 @@
 h1 {
```

Comment:
This seems to apply to all styles on the page as the stylesheet is loaded through PHP. I might be better to at some point to move those general styles to the vue components and make use of scoped styling.